server/src/table_notify.go

```diff
@@ -254,7 +254,8 @@ func (t *Table) NotifySpectatorsNote(order int) {
 		// the shadowed player
 		type Note struct {
 			Name string `json:"name"`
-			Note string `json:"note"`
+			Note string `json:"text"`
+			// ^^ TODO maybe the "Note" field should be renamed too
```

Comment:
should we rename it to text?

Reply:
In the server or the client? I assumed `text` in the client because it was clearer (so it's obviously a `string` and not a `SpectatorNote`, `CardNote`). For the server, it probably matters less.

Reply:
refactor it to be text on both server and client

Reply:
e.g. the name of the struct can't be different than the json representation, that is just confusing

Reply:
OK